autoscale/src/main/java/com/sequenceiq/periscope/controller/AutoScaleClusterCommonService.java

```diff
@@ -114,28 +117,39 @@ public Cluster setStopStartScalingState(Long clusterId, Boolean enableStopStartS
         Cluster cluster = clusterService.findById(clusterId);
         if (enableStopStartScaling != null && !cluster.isStopStartScalingEnabled().equals(enableStopStartScaling)) {
             cluster = clusterService.setStopStartScalingState(cluster, enableStopStartScaling);
-            processAutoscalingStateChanged(cluster);
         }
         return cluster;
     }
 
     public void deleteAlertsForClusterCrn(String stackCrn) {
         Cluster cluster = getClusterByCrnOrName(NameOrCrn.ofCrn(stackCrn));
+        deleteStoppedNodesIfPresent(cluster.getId());
         cluster = clusterService.deleteAlertsForCluster(cluster.getId());
-        processAutoscalingStateChanged(cluster);
+        processAutoscalingConfigChanged(cluster);
```

Comment:
This call, and the call in deleteAlertsForClusterName - will result in a message to the console saying "AutoScaling DISABLED". That isn't precise since AutoScaling is still enabled - there's just no policies defined (which in theory is disabled). Would be better to change the message to say - "All policies deleted" rather than "AutoScaling disabled" - since technically disabling autoscale is an independent operation.

Reply:
Addressed this. The message now says `All Autoscale policies deleted` when these delete endpoints are invoked.